android/app/src/main/java/org/fossasia/openevent/dbutils/RealmDataRepository.java

```diff
@@ -468,6 +469,29 @@ public Completable saveLocations(final List<Microlocation> locations) {
         return realm.where(Microlocation.class).findAllSorted("name");
     }
 
+    //Session Types Section
+
+    private void saveSessionTypesInRealm(List<SessionType> sessionTypes) {
+        Realm realm = Realm.getDefaultInstance();
+        realm.executeTransaction(realm1 -> realm1.insertOrUpdate(sessionTypes));
+        realm.close();
+    }
+
+    public Completable saveSessionTypes(final List<SessionType> sessionTypes) {
+        return Completable.fromAction(() -> {
+            saveSessionTypesInRealm(sessionTypes);
+            Timber.d("Saved Session Types");
+        });
+    }
+
+    public RealmResults<SessionType> getSessionTypes() {
```

Comment:
for this pr no. i am adding it for future reference.
or should i remove it then?